server/etcdmain/help.go

```diff
@@ -55,6 +55,8 @@ Member:
     Path to the dedicated wal directory.
   --snapshot-count '100000'
     Number of committed transactions to trigger a snapshot to disk.
+  --snapshot-catch-up-entries '5000'
```

Comment:
And please put it together with all other experimental flags,

```suggestion
  --experimental-snapshot-catch-up-entries '5000'
```